src/Fantomas.Tests/LetBindingTests.fs

```diff
@@ -1316,10 +1305,9 @@ let x =
        | _ ->
            if tcref.IsILTycon then
                tcref.ILTyconRawMetadata.CustomAttrs.AsArray
-               |> Array.exists
-                   (fun attr ->
-                       attr.Method.DeclaringType.TypeSpec.Name = typeof<TypeProviderEditorHideMethodsAttribute>
-                           .FullName)
+               |> Array.exists (fun attr ->
+                   attr.Method.DeclaringType.TypeSpec.Name = typeof<TypeProviderEditorHideMethodsAttribute>
+                       .FullName)
```

Comment:
This break of `.FullName` looks wrong to me.  Though it's an orthogonal problem

Reply:
This is unrelated to the changes in this PR, created https://github.com/fsprojects/fantomas/issues/1863 to follow up.